catalogue/webapp/components/WorkDetails/WorkDetails.js

```diff
@@ -5,18 +5,50 @@ import type { LicenseType } from '@weco/common/model/license';
 
 import NextLink from 'next/link';
 import styled from 'styled-components';
-import { font, spacing, grid, classNames } from '@weco/common/utils/classnames';
+import { font, spacing, classNames } from '@weco/common/utils/classnames';
 import { worksUrl } from '../../services/catalogue/urls';
 import { Fragment } from 'react';
 import SpacingComponent from '@weco/common/views/components/SpacingComponent/SpacingComponent';
 import Icon from '@weco/common/views/components/Icon/Icon';
 import Divider from '@weco/common/views/components/Divider/Divider';
 import CopyUrl from '@weco/common/views/components/CopyUrl/CopyUrl';
-import MetaUnit from '@weco/common/views/components/MetaUnit/MetaUnit2';
+import MetaUnit from '@weco/common/views/components/MetaUnit/MetaUnit';
+import Layout12 from '@weco/common/views/components/Layout12/Layout12';
 import Download from '../Download/Download';
 import DownloadBeta from '../Download/DownloadBeta';
 
-const StyledWorkDetailsSection = styled.div`
+type WorkDetailsSectionProps = {|
+  className?: string,
+  headingText?: string,
+  children: Node,
+|};
+
+const WorkDetailsSection = ({
+  className,
```

Comment:
I can't see this className being used anywhere, do we need it?

Reply:
Used here: https://github.com/wellcometrust/wellcomecollection.org/pull/4131/files#diff-315d33cd8dcf0d3cf36095070f4ea590R32